__tests__/app.test.js

```diff
@@ -34,4 +34,34 @@ describe("news api", () => {
       });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    describe("GET", () => {
+      test("200: returns an article object, containing author, title, article_id, body, topic, created_at & votes properties", () => {
```

Comment:
Very cool test. I would also make an assertion that article_id is definitely `1` :)

Reply:
Good idea, thanks Karla!